Description: Use int64_t instead of signed long int for 64-bit integer typedef
 On 32-bit architectures, longs are only 32 bits.  The resulting overflow
 was causing an infinite loop in the 0602ResultantFanProjection test.
Bug-Debian: https://bugs.debian.org/905300
Bug-Debian: https://bugs.debian.org/974558
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-11-12

(mjo: added missing include)

--- a/src/vektor.h
+++ b/src/vektor.h
@@ -6,11 +6,12 @@
 #include <assert.h>
 #include <algorithm>
 #include <complex>
+#include <cstdint>
 #include <stdio.h>
 
 using namespace std;
 
-typedef signed long int int64;
+typedef int64_t int64;
 
 void outOfRange(int i, int n);
 

